build.sbt

```diff
@@ -36,28 +32,27 @@ pomExtra := (
 
 licenses := Seq("Apache V2" -> url("http://www.apache.org/licenses/LICENSE-2.0.html"))
 
+publishTo := sonatypePublishToBundle.value
```

Comment:
we have to publish to the local target directory, then Bundle publish will actually push them